src/tribler-core/tribler_core/modules/tests/test_versioncheck.py

```diff
@@ -116,6 +120,19 @@ async def test_version_check_timeout(self):
         self.should_call_new_version_callback = False
         await self.check_version()
 
+    @timeout(10)
+    async def test_version_check_api_timeout(self):
+        await self.setup_version_server(json.dumps({'name': 'v1337.0'}))
+
+        # Setting the API response lag to 2 seconds and
+        # client's version checkout timeout of 1ms, version checks should fail returning False
```

Comment:
Is the checkout timeout in seconds or milliseconds (you are using both units)?

Reply:
Thanks for pointing out. It is in seconds. I'll update the comment.